README.md

```diff
@@ -314,6 +315,33 @@ ReactDOM.render(<Clock />, document.getElementById('react'))
 
 Try this live: <https://jsfiddle.net/arunoda/wm6romh4/>
 
+## Extending
```

Comment:
I get it. For your use case referring in the displayName.
It doesn't hurt if we add this to doc. I will add.
On 2016 ජන 19, අඟහ at ප.ව. 7.58 zVictor <notifications@github.com> wrote:

> In README.md
> <https://github.com/kadirahq/react-komposer/pull/18#discussion_r50117501>:
>
> > @@ -314,6 +315,33 @@ ReactDOM.render(<Clock />, document.getElementById('react'))
> >
> >  Try this live: <https://jsfiddle.net/arunoda/wm6romh4/>
> >
> > +## Extending
>
> It didn't work for me. I might be doing something wrong, though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kadirahq/react-komposer/pull/18/files#r50117501>.
>